webapp/STYLE_GUIDE.md

```diff
@@ -0,0 +1,157 @@
+# Webapp Style Guide
```

Comment:
Total nitpicking on my end, but I'm going to change this to "Web App" instead of "Webapp" because that's a stupid pet peeve of mine 😛